src/index.js

```diff
@@ -19,7 +20,7 @@ const store = createStore(counter);
 // where the store "lives".
 ReactDOM.render(
   <Provider store={store}>
-    <Counter />
+    <Counter  increment={store.dispatch({type: "INCREMENT"})} decrement={store.dispatch({type: 'DECREMENT'})}/>
```

Comment:
You can get rid of `increment={store.dispatch({type: "INCREMENT"})} decrement={store.dispatch({type: 'DECREMENT'})}`.